tools/pv_tree_generator/util.py

```diff
@@ -203,24 +228,26 @@ def _read_stat_var():
                 constraint_properties.append(val)
             else:
                 sv_dict[prop] = val
+        # prop_val keeps all the constraint pv pairs
         prop_val = {}
         for property in constraint_properties:
             if property not in sv_dict:
                 raise Exception('constraint property:{} not found in statistical'
                                 'variable with dcid: {}'.format(property, dcid))
             prop_val[property] = sv_dict[property]
-        if "measurementDenominator" in sv_dict:
-            prop_val["md"] = sv_dict["measurementDenominator"]
-        se = {}  # Super enum
+
+        # Super enum
+        se = {}
         if 'crimeType' in prop_val:
             v = prop_val.get('crimeType', '')
             if v in ['AggravatedAssault', 'ForcibleRape', 'Robbery',
                      'MurderAndNonNegligentManslaughter']:
                 se = {'crimeType': 'ViolentCrime'}
             elif v in ['MotorVehicleTheft', 'LarcenyTheft', 'Burglary']:
                 se = {'crimeType': 'PropertyCrime'}
+        # create the statsVar object
         sv = StatVar(sv_dict["populationType"], sv_dict["measuredProperty"],
```

Comment:
nit: new line for each argument when constructing StatVar

Reply:
done